### DISCUSSION TITLE
BatchSpanProcessorOptions 编译2.0.0的be，报错

### DISCUSSION THREAD

**Discussion by mhyeagle** [apache/doris#24339] [RFC]
ubuntu22.04
docker:  build-env-ldb-toolchain-latest
用最新的docker镜像编译2.0.0的代码，编译be时报错

/root/doris/apache-doris-2.0.0-src/be/src/util/telemetry/telemetry.cpp:93:42: error: variable has incomplete type 'trace_sdk::BatchSpanProcessorOptions'

**Comment by yagagagaga** [comment]
When Doris 2.0 is released, the compiled image `apache/doris:build-env-for-2.0` will also be released at the same time.  `docker: build-env-ldb-toolchain-latest` only applicable to the master branch.

**Reply by mhyeagle** [reply]
Thanks, i will try the 2.0 dev version

**Reply by CanGuan** [reply]
我在构建2.0.2-rc05也有类似的问题，用build-env-ldb-toolchain-latest / build-env-for-2.0 都会报错
